index.html

```diff
@@ -228,8 +231,10 @@ <h3 id="string_specific_direction">String-specific directional information</h3>
 <p class="advisement" id="bp-dir_field_based_metadata"><a class="self" href="#bp-dir_field_based_metadata">&#x200B;</a>If a resource-wide setting is available, specify field-based metadata to override the default. </p>
 <p>First-strong heuristics are ineffective when a default direction has been set for all strings, since  metadata overrides (intentionally) the value of the first-strong character. Therefore it is necessary to use explicitly provided field data to override the default. Even if an RLM character has been prepended to a string, the default metadata overrides it.</p>
 <p>The use of <a href="#metadata">metadata</a> for indicating base direction is also preferred, because it avoids requiring the consumer to interpolate the direction using methods such as <a href="#firststrong">first strong</a> or use of methods which require modification of the data itself (such as the <a href="#rlm">insertion of RLM/LRM markers</a> or <a href="#paired">bidirectional controls</a>).</p>
-<p class="issue">Schema languages, such as the RDF suite of specifications, have no in-built mechanism for associating base direction metadata with natural language string values.</p>
-<p class="issue">There is no built-in attribute for base direction in [[JSON-LD]]. There needs to be a corresponding built-in attribute (e.g. a <code class="kw" translate="no">dir</code>) or de facto convention for indicating document-level base direction.</p>
+<div class="note">
+	<p>Some schema languages, such as the RDF suite of specifications, have no built-in mechanism for associating base direction metadata with natural language string values. It is up to specifications that use these specifications to define structures and adopt best practices that result in clean interchange of language and direction metadata.</p>
+    <p>For example, [[JSON-LD]] provides a document-level base direction using the <code>@Context</code> mechanism and defines the <code>i18n namespace</code> as a non-standard RDF datatype for setting either/both language and base direction of a string.</p>
```

Comment:
See my comment above. RDF does not have a built-in mechanism, but now has an 'extension', as proposed in the JSON-LD specification, to use i18n specific terms. That can be used by any user of RDF, regardless of serialization...

Reply:
See if the new revision captures it. 

I know I need to do more work here--adding documentation and examples later in the doc, etc.